R/limits.r

```diff
@@ -46,6 +48,25 @@
 #' ggplot(big, aes(mpg, wt, colour = factor(cyl))) +
 #'   geom_point() +
 #'   lims(colour = c("4", "6", "8"))
+#'
+#' # There are two ways of setting the axis limits: with limis or
+#' # with coordinate systems.  They work in two rather different ways.
+#'
+#'  p <- ggplot(mtcars, aes(disp, wt)) +
```

Comment:
Can you please check the indenting here and below? There should be two extra spaces for the lines after `+`, and `p` needs one less space.